### PR TITLE
ktx 4.3.2 (new formula)

### PR DESCRIPTION
**Pull request by pthariensflame**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

### CODE REVIEW

File: Formula/k/ktx.rb [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    "BSD-1-Clause",
+    "BSD-2-Clause",
+    "BSD-3-Clause",
+    "BSL-1.0",
+    "CC-BY-3.0",
+    "CC-BY-4.0",
+    "CC0-1.0",
+    "MIT",
+    "Zlib",
+    :cannot_represent, # Cesium Trademark Terms, ETCSLA, HI Trademark, Kodak, PNGSuite
```
Comment (SMillerDev) [inline Formula/k/ktx.rb:17]: This sounds like it might not be open source. Are there other package managers shipping this software?

**Review by SMillerDev — CHANGES_REQUESTED** [review]

File: Formula/k/ktx.rb (hunk 1 above)
Comment (pthariensflame) [reply Formula/k/ktx.rb:17]: Yes, although a cursory search only found one:
https://www.freshports.org/graphics/khronos-texture/

The licenses listed are taken from the files in the repository; glancing over them didn’t reveal anything specifically closed-source, but IANAL.

Comment (SMillerDev) [reply Formula/k/ktx.rb:17]: Let's make the whole thing `:cannot_represent` it seems better than this whole list of things that may or may not apply

Comment (pthariensflame) [reply Formula/k/ktx.rb:17]: What should I put in the comment? Just “see this repo URL”?

Comment (SMillerDev) [reply Formula/k/ktx.rb:17]: Something like this?
> Too complicated to represent in the formula

File: Formula/k/ktx.rb [hunk 2]
```diff
@@ -0,0 +1,32 @@
+class Ktx < Formula
+  desc "Khronos Texture Library and Tools"
+  homepage "https://github.com/KhronosGroup/KTX-Software"
+  url "https://github.com/KhronosGroup/KTX-Software/archive/refs/tags/v4.3.2.tar.gz"
+  sha256 "74a114f465442832152e955a2094274b446c7b2427c77b1964c85c173a52ea1f"
+  license :cannot_represent # too complicated to represent in the formula
+  # no HEAD URL because it would require git-lfs and Homebrew doesn't support using that
```
Comment (SMillerDev) [inline Formula/k/ktx.rb:7]: Head URLs are not required. We can just leave them out
```suggestion
```

File: Formula/k/ktx.rb [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+  depends_on "cmake" => :build
+  depends_on "ninja" => :build
+
+  def install
+    # fix assorted rpath and version reporting issues
+    inreplace "tools/CMakeLists.txt",
+              "INSTALL_RPATH \"@executable_path;/usr/local/lib\"",
+              "INSTALL_RPATH \"@executable_path;#{lib}\""
+    inreplace "mkversion", "DEF_VER=v4.0", "DEF_VER=v#{version}"
```
Comment (SMillerDev) [inline Formula/k/ktx.rb:22]: Can you report this upstream?

**Pull request closed without merging**